File: JavaBasic/146_ascii_values.java
// 146	146_ascii_values.java	Print the ASCII (char code) value of each character in a string.
public class _146_ascii_values {
    public static void main(String[] args) {
        String str = "Hello";
        for (char c : str.toCharArray())
            System.out.println("'" + c + "' => " + (int)c);
    }
}

File: JSBASIC/146_ascii_values.js
// 146	146_ascii_values.js	Print the ASCII (char code) value of each character in a string.

let str = "Hello";
console.log("String:", str);
for (let i = 0; i < str.length; i++) {
    console.log("'" + str[i] + "' => " + str.charCodeAt(i));
}
